roles/openshift_node_upgrade/tasks/main.yml

```diff
@@ -114,12 +110,21 @@
     command: swapoff --all
 
   - name: Remove swap entries from /etc/fstab
+    replace:
+      dest: /etc/fstab
+      regexp: '(^[^#].*swap.*)'
+      replace: '# \1'
+      backup: yes
+
+  - name: Add notice about disabling swap
     lineinfile:
       dest: /etc/fstab
-      regexp: 'swap'
-      state: absent
+      line: '# OpenShift-Ansible Installer disabled swap per overcommit guidelines'
+      state: present
 
-  when: swap_result.stdout_lines | length > 0
+  when:
+  - swap_result.stdout_lines | length > 0
+  - openshift_disable_swap | default(true)
```

Comment:
Out of curiosity I checked to see what the failures would be if we forced `true` or `false` for `indent-sequences`

yamllint errors with:
`indent-sequences: false`: 200
`indent-sequences: true`: 1345